src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -44,15 +45,16 @@
     (i18n/label :t/opensea)]])
 
 (def tabs-data
-  [{:id                  :overview
-    :label               (i18n/label :t/overview)
-    :accessibility-label :overview-tab}
-   {:id                  :activity
-    :label               (i18n/label :t/activity)
-    :accessibility-label :activity-tab}
-   {:id                  :about
-    :label               (i18n/label :t/about)
-    :accessibility-label :about-tab}])
+  (let [base-tabs    [{:id                  :overview
+                       :label               (i18n/label :t/overview)
+                       :accessibility-label :overview-tab}
+                      {:id                  :about
+                       :label               (i18n/label :t/about)
+                       :accessibility-label :about-tab}]
+        activity-tab [{:id                  :activity
+                       :label               (i18n/label :t/activity)
+                       :accessibility-label :activity-tab}]]
+    (concat base-tabs (if config/wallet-collectible-actvity-enabled? activity-tab []))))
```

Comment:
tbh I don't think we need this. Let's just delete the activity tab for now and keep it simple 👍